### ISSUE TITLE
The Biance contract interface has expired

### ISSUE THREAD

**Issue by KING627FFA** [ccxt/ccxt#19013]
### Operating System

windows

### Programming Languages

Python

### CCXT Version

__version__ = '4.0.76'

### Description

ccxt.base.errors.ExchangeError: binance {"code":-5000,"msg":"GET /fapi/v1/balance is retired, please use GET /fapi/v2/balance"}

### Code

```
  

```
money = binance.fapiprivate_get_balance()
print(money)

**Comment by samgermain** [comment]
Yes, you should use the new v2 implicit method, or save yourself the trouble and use the [unified `fetchBalance` method,](https://docs.ccxt.com/#/?id=account-balance) which we update to deal with version changes

**Comment by ttodua** [comment]
this issue was already fixed in the past.
